src/Workspaces/VisualBasicTest/Formatting/FormattingTests.vb

```diff
@@ -833,21 +833,21 @@ End Class</Code>
             Dim expected = <Code>Class C
     Sub Method(Optional ByVal i As Integer = 1)
         Dim aa = 1 + _
-2 + _
-3
+                     2 + _
+                         3
     End Sub
 End Class</Code>
 
             Await AssertFormatLf2CrLfAsync(code.Value, expected.Value)
         End Function
 
         <Fact, Trait(Traits.Feature, Traits.Features.Formatting)>
-        Public Async Function LineContinuation4() As Task
+        Public Async Function LineContinuation4___() As Task
             Dim code = <Code>Class C
     Sub Method(Optional ByVal i As Integer = 1)
                     Dim aa = 1 + _
             _
-                                          _
+                                          _' Comment 1
```

Comment:
please don't change the inputs of existing tests.  Instead, add a new test with the different input.  By doing this, i can't see what the impact of your code change is on hte original input of the original test.

Reply:
Yes I realized I needed to do that. Some of the old tests will fail but I think for now that is OK.

Reply:
thanks!